### PR TITLE
add videos page

### PR DESCRIPTION
**Pull request by hzoo**
Fixes https://github.com/babel/website/issues/27
Ref https://twitter.com/left_pad/status/918654512377851904

Just a start, can make new issues to add more.

Added some from https://www.youtube.com/playlist?list=PLUjGYjn761TW_j4HlBcq_qnJoaSMTlJOH, https://www.youtube.com/watch?v=Q_ncaTYEizc

### CODE REVIEW

File: docs/community/videos.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+---
+layout: docs
+title: Videos
+description: Videos and podcasts about Babel and it's underlying concepts
```
Comment (existentialism) [inline docs/community/videos.md]: Nit: its

File: docs/community/videos.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+permalink: /docs/community/videos/
+---
+
+## From The Core Team
+
+### [JQuerySF 2015] Compiling JavaScript
```
Comment (existentialism) [inline docs/community/videos.md]: Nit: JQuerySF -> jQuerySF

**Comment by xtuc** [comment]
I have https://youtu.be/rpb9iOKyO30 (I don't like it!). I'm waiting for the video from FableConf, probably available soon.

**Comment by existentialism** [comment]
Think this is a good start, I'm imagining we'll probably convert this page to a grid of videos grouped by topic or something shortly?

**Comment by xtuc** [comment]
@existentialism yes, I think we should group them by date (only year maybe)

**Pull request merged**